src/main/java/org/fundacionjala/coding/ana/DnaStrand.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.ana;
+
+import java.util.HashMap;
+
+/**
+ * DNA.
+ *
+ * @author Ana Maria Mamani Zenteno.
+ */
+public class DnaStrand {
+    /**
+     * method for the complement of DNA.
+     *
+     * @param adn string of into.
+     * @return  a string.
+     */
+    public String makeComplement(String adn) {
+        StringBuffer getComplement = new StringBuffer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)